src/OrchardCore.Modules/OrchardCore.Users/Views/Admin/Index.cshtml

```diff
@@ -1,58 +1,47 @@
-                    <a class="dropdown-item" href="#">@T["Disable"]</a>
-                    <a class="dropdown-item" href="#">@T["Send challenge E-mail"]</a>
-                    <a class="dropdown-item" href="#">@T["Delete"]</a>
+<form asp-action="Index" method="get">
+    <fieldset class="filter-options">
+        <div class="card mb-3">
+            <div class="card-body bg-light p-3">
+                <div class="row">
+                    <div class="form-group col-xl-6 mb-xl-n1">
+                        <div class="input-group input-group-sm">
+                            <div class="input-group-prepend">
+                                <span class="input-group-text bg-secondary text-light">@T["Filter"]</span>
+                            </div>
+                            <input id="search-box" asp-for="Options.Search" class="form-control" autofocus="autofocus" />
+                        </div>
+                    </div>
+                    <div class="form-group col-xl-3 mb-xl-n1">
+                        <div class="input-group input-group-sm">
+                            <button type="submit" class="btn btn-sm btn-secondary">@T["Search"]</button>
+                        </div>
+                    </div>
                 </div>
             </div>
-        </li>*@
-        @*@Html.TextBoxFor(m => m.Options.Search, new { @class = "text" })
-            <label for="filterResults">@T("Filter:")</label>
-            <select id="filterResults" name="@Html.NameOf(m => m.Options.Filter)">
-                @Html.SelectOption(Model.Options.Filter, UsersFilter.All, T("All Users").ToString())
-                @Html.SelectOption(Model.Options.Filter, UsersFilter.Approved, T("Approved Users").ToString())
-                @Html.SelectOption(Model.Options.Filter, UsersFilter.Pending, T("Pending Users").ToString())
-                @Html.SelectOption(Model.Options.Filter, UsersFilter.EmailPending, T("Pending Emails").ToString())
-            </select>
-            <label for="sortResults">@T("Sort by:")</label>
-            <select id="sortResults" name="@Html.NameOf(m => m.Options.Order)">
-                @Html.SelectOption(Model.Options.Order, UsersOrder.Name, T("Name").ToString())
-                @Html.SelectOption(Model.Options.Order, UsersOrder.Email, T("Email").ToString())
-                @Html.SelectOption(Model.Options.Order, UsersOrder.CreatedUtc, T("Creation Time").ToString())
-                @Html.SelectOption(Model.Options.Order, UsersOrder.LastLoginUtc, T("Last Login Time").ToString())
-            </select>*@
-    </ul>
-    <form asp-action="Index" method="get" class="form-inline">
-        <input id="search-box" asp-for="Options.Search" class="form-control mr-sm-2 my-2 my-lg-0" type="text" placeholder="@T["Search"]" autofocus="autofocus">
-        <button type="submit" class="btn btn-secondary">@T["Search"]</button>
-    </form>
-</nav>
+        </div>
+    </fieldset>
+</form>
 
-@using (Html.BeginForm())
```

Comment:
You are removing the form. does it still work?

Reply:
Yep, coz if you look to the search bar it surroundings with a form, may be the deleted form was added for a reason in the past and no one removed